AirCasting/CreateSessionViews/CreateSessionDetails/CreateSessionDetailsViewModel.swift

```diff
@@ -24,6 +25,10 @@ class CreateSessionDetailsViewModel: ObservableObject {
         self.baseURL = baseURL
     }
     
+    func onSessionNameChange() {
+        (sessionName != "") ? (shouldShowError = false) : (shouldShowError = true)
```

Comment:
a bit overcomplicated, could've been:

```Swift
shouldShowError = (sessionName == "")
```

But there is no need for this method `onSessionNameChange` at all here. It can be done purely inside viewmodel, just change `shouldShowError` to a computed property instead:
``` Swift
class CreateSessionDetailsViewModel: ObservableObject {
    @Published var sessionName = ""
    var shouldShowError { sessionName.isEmpty }
}
```

Since `shouldShowError` is only read whenever `sessionName` changes we don't need to worry about `objectWillChange.send()` because it will get called each time `sessionName` changes thus forcing any `shouldShowError` reads too.

Reply:
Thank you, looks so much better!